opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -39,22 +39,34 @@ fn init_logs() -> Result<sdklogs::LoggerProvider, opentelemetry::logs::LogError>
     opentelemetry_otlp::new_pipeline()
         .logging()
         .with_resource(RESOURCE.clone())
-        .with_exporter(http_exporter().with_endpoint("http://localhost:4318/v1/logs"))
+        .with_exporter(
+            http_exporter()
+                .with_protocol(Protocol::HttpBinary)
+                .with_endpoint("http://localhost:4318/v1/logs"),
+        )
         .install_batch(opentelemetry_sdk::runtime::Tokio)
 }
 
 fn init_tracer_provider() -> Result<sdktrace::TracerProvider, TraceError> {
     opentelemetry_otlp::new_pipeline()
         .tracing()
-        .with_exporter(http_exporter().with_endpoint("http://localhost:4318/v1/traces"))
+        .with_exporter(
+            http_exporter()
+                .with_protocol(Protocol::HttpBinary)
```

Comment:
I am not sure if I follow the PR's intent.. The title says it is updating example to show json payload, but actual changes does not indicate so.. Could you please clarify the intent?

Reply:
I think we can just add a comment here (and Readme.md) on how to switch protocol from `HttpBinary` to `HttpJson`. And default can remain binary.